ecs-agent/acs/session/task_manifest_responder.go

```diff
@@ -0,0 +1,185 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package session
```

Comment:
Believe we'll need to rerun `go mod tidy` and `go mod vendor` in agent module to pick up the changes from the latest revision of this PR.

Reply:
ack

Reply:
Seems like there is still a diff in the vendored files in agent module and what is present in ecs-agent module. Can we run `go mod tidy` and `go mod vendor` in agent module to pick up the changes from ecs-agent module?